src/tests/unit/tests/debug-tools/components/debug-tools-nav.test.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { DebugToolsNavActionCreator } from 'debug-tools/action-creators/debug-tools-nav-action-creator';
+import { DebugToolsNav, DebugToolsNavProps } from 'debug-tools/components/debug-tools-nav';
+import { ToolsNavKey } from 'debug-tools/stores/debug-tools-nav-store';
+import { shallow } from 'enzyme';
+import { INavLink, Nav } from 'office-ui-fabric-react';
+import * as React from 'react';
+import { Mock, Times } from 'typemoq';
+
+describe('DebugToolsNav', () => {
+    it('renders', () => {
+        const props: DebugToolsNavProps = {
```

Comment:
Feels like you can use a beforeEach and share the props.

Reply:
update